tools/encodings/GF Glyph Sets/Cyrillic/GF-cyrillic-pro_unique-glyphs.nam

```diff
@@ -1,4 +1,5 @@
 # Created by Alexei Vanyashin 2016-27-06
+#$ include GF-cyrillic-plus_unique-glyphs.nam
 0x048A  Ҋ uni048A
```

Comment:
@graphicore These lines should be reformatted to `0x | Cha | Unicode Description` format using your latest namelist.py file